docs/howto/features/buckets.md

```diff
@@ -106,6 +106,35 @@ user_buckets = {
 }
 ```
 
+## Enabling Requester Pays flag on buckets
```

Comment:
We currently don't have this functionality at all, so we should turn this into a feature request issue and remove this section. See https://github.com/2i2c-org/infrastructure/pull/3431/files#r1498313106 for some more info.